### PR TITLE
Support to get client tls version

### PR DESCRIPTION
**Pull request by pyama86**
## Pull-Request Check List
hi! ry sann!

SSLハンドシェーク中にTLSのバージョンを取得できるようにしました。これによって古いクライアントからの接続要求時に処理を分岐するなどの実装が可能になります。

- [x] Add patches into `src/`.
- [x] Add test into `test/`. Please see about [test docs](https://github.com/matsumotory/ngx_mruby/tree/master/docs/test).
- [x] Add docs into `docs/` if you change the features such as [build system](https://github.com/matsumotory/ngx_mruby/tree/master/docs/install), [Ruby methods, class](https://github.com/matsumotory/ngx_mruby/tree/master/docs/class_and_method) and [nginx directives](https://github.com/matsumotory/ngx_mruby/tree/master/docs/directives).

### CODE REVIEW

File: src/http/ngx_http_mruby_ssl.c [hunk 1]
```diff
@@ -127,6 +145,11 @@ static mrb_value ngx_mrb_ssl_local_port(mrb_state *mrb, mrb_value self)
   mrb_raise(mrb, E_RUNTIME_ERROR, "ngx_mrb_ssl_local_port doesn't support");
 }
 
+static mrb_value ngx_mrb_ssl_tls_version(mrb_state *mrb, mrb_value self)
+{
+  mrb_raise(mrb, E_RUNTIME_ERROR, "ngx_mrb_ssl_ssl_tls_version doesn't support");
```
Comment (udzura) [inline src/http/ngx_http_mruby_ssl.c]: TYPO: ssl_ is double

File: docs/class_and_method/README.md [hunk 2]
```diff
@@ -276,16 +276,34 @@ alias of Nginx::SSL.errlogger
...
+mruby_ssl_handshake_handler_code '
+  ssl = Nginx::SSL.new
+  ssl.certificate = "__NGXDOCROOT__/#{ssl.servername}.crt"
+  ssl.certificate_key = "__NGXDOCROOT__/#{ssl.servername}.key"
+  Userdata.new.ssl_local_port = ssl.local_port
+';
+
 location /local_port {
-     mruby_content_handler_code "Nginx.rputs Nginx::SSL.new.local_port.to_s";
+     mruby_content_handler_code "Nginx.rputs Userdata.new.ssl_local_port.to_s";
```
Comment (matsumotory) [inline docs/class_and_method/README.md]: Oh! It's my mistake. Thanks!

Comment (pyama86) [reply docs/class_and_method/README.md]: I'll fix this much, brothers!

**Comment by matsumotory** [comment]
lgmt! I'll merge this after passing CI, and release new version.

**Comment by matsumotory** [comment]
Thanks always!

**Pull request merged**